gitpod-io/website
# :sparkles: New `/pricing` page

## Description



This PR updates `/pricing` page

## Related Issue(s)



Fixes MKWEB-94

## How to test



- Open preview
- See [`/pricing`](https://deploy-preview-3851--gitpod-kumquat.netlify.app/pricing) page :eyes:

## Release QA Checklist ✔️



<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing


## [Priority: High] Content Testing


## [Priority: Medium] Performance Testing


## [Priority: Medium] Accessibility Testing


By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies, data storing, or analytics.


</details>


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3851"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
